tests/unit/test_monitor.py

```diff
@@ -0,0 +1,38 @@
+from typing import Any, Tuple
+
+import pytest
+
+MONITOR_API_TEST_CASES: Tuple[Tuple[str, str, Any], ...] = (
+    ("HEAD", "/__lbheartbeat__", None),
+    ("GET", "/__lbheartbeat__", None),
+    ("HEAD", "/__heartbeat__", None),
+    ("GET", "/__heartbeat__", None),
+)
+
+
+@pytest.mark.parametrize("method,path,params", MONITOR_API_TEST_CASES)
+def test_unauthorized_api_call_fails(
+    anon_client, example_contact, method, path, params
+):
+    """Calling the API without credentials fails."""
+    _map = {  # METHOD: ( {_REQUEST_PARAMS} , {_EXPECTED_RESULTS}
+        "GET": ({"method": "GET", "url": path, "params": params}, {200}),
+        "HEAD": ({"method": "HEAD", "url": path}, {200}),
+    }
+    if method in _map.keys():
+        _request, _results = _map.get(method)
+        resp = anon_client.request(**_request)
+        assert resp.status_code in _results
+
+
+@pytest.mark.parametrize("method,path,params", MONITOR_API_TEST_CASES)
+def test_authorized_api_call_succeeds(client, example_contact, method, path, params):
+    """Calling the API without credentials fails."""
+    _map = {  # METHOD: ( {_REQUEST_PARAMS} , {_EXPECTED_RESULTS}
+        "GET": ({"method": "GET", "url": path, "params": params}, {200}),
+        "HEAD": ({"method": "HEAD", "url": path}, {200}),
+    }
+    if method in _map.keys():
+        _request, _results = _map.get(method)
+        resp = client.request(**_request)
+        assert resp.status_code in _results
```

Comment:
No, this copies the API tests without making any of the changes needed to test the monitoring endpoints. The test is called ``_fails`` but tests that they return 200s, tests with an authenticated client even though API credentials are not needed, and for some reason passes ``{"params": None}`` for a few cases.

Let's do this:

```suggestion
@pytest.mark.parametrize("method", ("HEAD", "GET"))
@pytest.mark.parametrize("path", ("/__lbheartbeat", "/__heartbeat"))
def test_monitoring_endpoints(anon_client, method, path):
    """Monitoring endpoints can be called without credentials"""
    resp = anon_client.request(method, path)
    assert resp.status_code == 200
```

Reply:
but, the existing tests do pass and test the new methods.

I'm touching the monitoring tests for issue #153, I can make my changes then if you want to merge and move on to the next issue.